From 53253d953f3797cc4ed07628fe82f86f1b20061c Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Wed, 24 May 2017 11:50:21 +0800
Subject: [PATCH] ARM: dts: rockchip: add memory node for rk3229-echo-v10

when using upstream u-boot load kernel, reserves memory from early
allocator will fail if memory node is not specified.

===============================
Starting kernel ...

[    0.000000] Booting Linux on physical CPU 0xf00
[    0.000000] Initializing cgroup subsys cpuset
[    0.000000] Initializing cgroup subsys cpu
[    0.000000] Initializing cgroup subsys cpuacctdd
...
[    0.000000] earlycon: Early serial console at MMIO32 0x11030000 (options '')
[    0.000000] bootconsole [uart0] enabled
[    0.000000] cma: Failed to reserve 16 MiB
[    0.000000] Memory policy: Data cache writealloc

This patch fix it.

Change-Id: I6a3c6b1e210bbc9a5240503ab7bf5ddab89910ba
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-echo-v10.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rk3229-echo-v10.dts b/arch/arm/boot/dts/rk3229-echo-v10.dts
index 2b5354fb7490..c71ef171238e 100644
--- a/arch/arm/boot/dts/rk3229-echo-v10.dts
+++ b/arch/arm/boot/dts/rk3229-echo-v10.dts
@@ -50,6 +50,11 @@
 	model = "RK3229 Echo Board V1.0";
 	compatible = "rockchip,rk3229-echo", "rockchip,rk3229";
 
+	memory@60000000 {
+		device_type = "memory";
+		reg = <0x60000000 0x40000000>;
+	};
+
 	vcc_host: vcc-host-regulator {
 		compatible = "regulator-fixed";
 		enable-active-high;
-- 
2.35.3

